> /Users/runner/runners/2.164.8/work/1/s/src/Components/Blazor/testassets/HostedInAspNet.Server/HostedInAspNet.Server.csproj : warning NU1602: Microsoft.AspNetCore.Blazor does not provide an inclusive lower bound for dependency Mono.WebAssembly.Interop. An approximate best match of Mono.WebAssembly.Interop 0.5.0 was resolved.

> Services/WebAssemblyJSRuntime.cs(9,50): error CS0012: The type 'JSInProcessRuntimeBase' is defined in an assembly that is not referenced. You must add a reference to assembly 'Microsoft.JSInterop, Version=0.5.0.0, Culture=neutral, PublicKeyToken=null'. [/Users/runner/runners/2.164.8/work/1/s/src/Components/Blazor/Blazor/src/Microsoft.AspNetCore.Blazor.csproj]

@pranavkm should I instead have pinned the dependency on Mono.WebAssembly.Interop?

--- a/eng/Versions.props
+++ b/eng/Versions.props
@@ -204,6 +204,7 @@
     <!-- Packages developed by @aspnet, but manually updated as necessary. -->
     <LibuvPackageVersion>1.10.0</LibuvPackageVersion>
     <MicrosoftAspNetWebApiClientPackageVersion>5.2.6</MicrosoftAspNetWebApiClientPackageVersion>
+    <MonoWebAssemblyInteropPackageVersion>3.1.1-preview4.19614.4</MonoWebAssemblyInteropPackageVersion>
     <!-- Partner teams -->
     <MicrosoftAzureKeyVaultPackageVersion>2.3.2</MicrosoftAzureKeyVaultPackageVersion>
     <MicrosoftAzureStorageBlobPackageVersion>10.0.1</MicrosoftAzureStorageBlobPackageVersion>

